uplink/types.py

```diff
@@ -269,6 +269,19 @@ def modify_request(self, request_builder, value):
 
 
 class Query(NamedArgument):
+    """Sets one dynamic query parameter.
+
+    Queries the API using it's defined argument. You can include it in
+    your function methody by using the format: <query argument>:uplink.Query.
```

Comment:
This paragraph looks good overall. Here, I would replace the first sentence with "This annotation turns argument values into URL query parameters". Also, there's a typo on `methody` in the second sentence (the word `methody` has a nice ring to it, though 😄).